vsgenstubs/_vapoursynth.part.pyi

```diff
@@ -242,21 +240,21 @@ class Environment:
 class EnvironmentPolicyAPI:
     def wrap_environment(self, environment_data: EnvironmentData) -> Environment: ...
     def create_environment(self) -> EnvironmentData: ...
-    def unregister_policy(self) -> None: ...
+    def unregister_policy(self) -> typing.NoReturn: ...
 
 class EnvironmentPolicy:
-    def on_policy_registered(self, special_api: EnvironmentPolicyAPI) -> None: ...
-    def on_policy_cleared(self) -> None: ...
-    def get_current_environment(self) -> typing.Optional[EnvironmentData]: ...
+    def on_policy_registered(self, special_api: EnvironmentPolicyAPI) -> typing.NoReturn: ...
+    def on_policy_cleared(self) -> typing.NoReturn: ...
+    def get_current_environment(self) -> typing.Optional[EnvironmentData]: ...  # these functions currently don't return anything, they raise a `NotImplementedError` ?
```

Comment:
This method is intended to be subclassed by the explicit policy implementation

Reply:
See https://github.com/vapoursynth/vsrepo/pull/125#discussion_r435502644, I removed these changes.